src/index.js

```diff
@@ -45,6 +45,15 @@ export default class ESjs {
     this.indexDoc(doc);
   }
 
+  removeDoc(id) {
+    if (!(id in this.docs)) {
```

Comment:
Small nitpick, but it's generally preferred to use `Object.hasOwnProperty` for this kind of check. The `in` operator traverses the object prototype which can sometimes lead to unexpected behavior

Reply:
Yeah I know, but this is so much cleaner to read! In all seriousness, I thought about that, but the key in object pattern is used elsewhere in this file (e.g. getNode) so I went for consistency.

Reply:
Agreed, looks much nicer :). If we're already using it in here, NBD